content/docs/1.8.0/nodes-and-volumes/volumes/create-volumes.md

```diff
@@ -51,6 +52,7 @@ When the Pod is deployed, the Kubernetes master will check the PersistentVolumeC
     ```
 
     In particular, starting with v1.4.0, the parameter `mkfsParams` can be used to specify filesystem format options for each StorageClass.
+    Starting with v1.8.0, the parameter `backupTargetName` can be used to specify the backup target for creating a backup.
```

Comment:
What is the default value if `backupTargetName` is not specified?

```suggestion
    Starting with v1.8.0, the parameter `backupTargetName` can be used to specify the backup target for creating a backup. If `backupTargetName` is not specified, ....
```

Reply:
Added. Thanks.